app/src/main/kotlin/com/wire/android/ui/home/whatsnew/WhatsNewItem.kt

```diff
@@ -138,3 +145,16 @@ fun previewFileRestrictionDialog() {
         )
     }
 }
+
+@PreviewMultipleThemes
+@Composable
+fun previewFileRestrictionDialogPlaceholder() {
```

Comment:
Lint will complain about preview instead of Preview

Reply:
Sure, I  only copied the existing one and missed that they have that names